topics/statistics/tutorials/loris_model/tutorial.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: tutorial_hands_on
+level: Intermediate
+title: Building the LORIS LLR6 PanCancer Model Using Galaxy-PyCaret Tool
+zenodo_link: https://zenodo.org/records/13885908
+questions:
+- How can I reproduce the LORIS LLR6 model published by Chang et al., 2024?
+- Which tools can I use in Galaxy to obtain a logistic regression model?
+- How can I evaluate the model to confirm its performance?
+objectives:
+- Use a large dataset of immune checkpoint blockade (ICB)-treated and non-ICB-treated patients across 18 solid tumor types, encompassing a wide range of clinical, pathologic and genomic features to build a Machine Learning Model.
+- Build a Machine Learning model using PyCaret in Galaxy.
+- Evaluate the model for robustness by comparing it with the original LORIS LLR6 model published by Chang et al., 2024.
```

Comment:
The LORIS LLR6 model should be discussed in the tutorial. Details such as model architecture, features etc should be mentioned. It would improve the tutorial.

Reply:
Thank you. I have included more information regarding the LLR6 model.